app/views/public_assets/_form.html.erb

```diff
@@ -41,5 +41,23 @@
     <%= form.select :validate_by, ["size", "version"], { selected: @public_asset.validate_by }, { class: "govuk-select" } %>
   </div>
 
+  <% if @public_asset.validate_by_version? %>
+    <div class="govuk-form-group">
+      <%= form.label :hosted_version_regex, class: "govuk-label govuk-label--m" %>
+      <div id="size-hint" class="govuk-hint">
+        Please enter a regex which has a capture group for the version number of the GOV.UK hosted JavaScript file.
```

Comment:
Just checking is "GOV.UK hosted JavaScript file" the correct term? Just wondering since that LUX script is on a CDN. Is that still a GOV.UK hosted JS file?

Reply:
Also this is a non blocking comment, feel free to merge it

Reply:
No, you're right - I think `GOV.UK maintained JavaScript file` is probably more accurate. I'll change it. Thanks @AshGDS 👍

Reply:
Fixed up